### DISCUSSION TITLE
有点饿了，省吃俭用半年终于也是玩了。

### DISCUSSION THREAD

**Discussion by YDwang1459** [share121/inter-knot#4981] [其他]
三国杀是世界上最好玩的游戏，新手开局就送五虎上将，不仅礼包多，任务也不难，新手7天福利也非常好，玩法众多，可以是卡牌游戏，还可以是射击游戏，排位机制十分平衡，非常适合大众游玩，是一款非常良心的游戏。
![Screenshot_20240916_152003_com sgshd nearme gamecenter](https://github.com/user-attachments/assets/3c48f448-1ad2-4c8b-baea-e6234df22861)

**Comment by frisk666666** [comment]
说得好，这就去给打差评！
